Modules/_xxsubinterpretersmodule.c

```diff
@@ -1167,6 +1167,8 @@ static PyObject *ChannelNotFoundError;
 static PyObject *ChannelClosedError;
 static PyObject *ChannelEmptyError;
 static PyObject *ChannelNotEmptyError;
+static PyObject *ChannelReleasedError;
+static PyObject *NotReceivedError;
```

Comment:
What are these for?

Reply:
These Exceptions I just added their lower level support.

Reply:
We don't use them anywhere.  They aren't in the PEP anymore either.